src/common/performance.js.flow

```diff
@@ -1,3 +0,0 @@
-declare export class performance {
```

Comment:
Why do we change `.js.flow` to `.js` here ? It's definitely not JS and flow given `.js.flow` a precedence over `.js` during imports.